app/views/computacenter/home/show.html.erb

```diff
@@ -27,5 +27,11 @@
     </h2>
 
     <p class="govuk-body">View details of schools with that can place orders.</p>
+
+    <h2 class="govuk-heading-m">
+      <%= govuk_link_to 'Support home', support_home_path %>
+    </h2>
```

Comment:
Is it worth adding logic to hide these links if a CC user did not have read access? I'd be happy to leave this to your judgement - it might be worth waiting until there's a need. 

```suggestion
    <% if policy(:support).readable? %>
      <h2 class="govuk-heading-m">
        <%= govuk_link_to 'Support home', support_home_path %>
      </h2>
    <% end %>
```

Reply:
Sure, done in a9a848f. I may do a follow-up PR to add authorisation/pundit into the CC area... Ideally we'd be able to push it down to the `ApplicationController` at some point.